jenkinsfiles/cordova_jenkinsfile.groovy

```diff
@@ -1,14 +1,72 @@
 /**
 * Cordova Jenkinsfile
 */
-node('android') {
-   stage 'Checkout'
-	 checkout scm
 
-	 stage 'Prepare'
-	 cordova platform add android
-	 cordova prepare
+def platform = params?.PLATFORM?.trim()
+def buildType = params?.BUILD_CONFIG?.trim()
+def keystore_path = "/opt/android-sdk-linux/android.debug"
```

Comment:
I think we can use the ANDROID_HOME env var in this string: `"${env.ANDROID_HOME}/android.debug"`